components/docs-chef-io/content/automate/ha_inplace_migration.md

```diff
@@ -28,11 +28,35 @@ This page explains the In-Place migration of A2HA to Automate HA. This migration
 ## Prerequisites
 
 - A healthy state of the A2HA cluster to take fresh backup.
-
 - A2HA is configured to take backup on a mounted network drive (location example: /mnt/automate_backup).
-
 - Availability of 60% of space.
 
+## Capture information about the current A2HA instance
+
+In order to verify the migration is completed successfully we'll need to capture some information about the current installation. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.
```

Comment:
Replace it with: We'll need to capture information about the current installation to verify the migration is completed successfully. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.